### PR TITLE
CASMINST-3016: Clarify Kubernetes NCNs in topics and add glossary entry csm 1.2

### PR DESCRIPTION
**Pull request by haroldlongley**
CASMTRIAGE-2096: Clarify Kubernetes NCNs in topics and add glossary entry csm 1.2

### CODE REVIEW

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
Friendly housekeeping note! This needs to be rebased/cherry-picked against the new main branch before it can be reviewed. Or (given the age of this PR), if the change is no longer needed or applicable, please close this badboy.

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
This is actually tied to https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-3016
The JIRA linked here routes to Cray JIRA and the JIRA has moved, so the link is broken.

File: glossary.md [hunk 1]
```diff
@@ -143,6 +143,12 @@ management VLAN is used for customer access and administration of hardware.
 
 The High Speed Network (HSN) in an HPE Cray EX system is based on the Slingshot switches.
 
+### Kubernetes NCNs
+
+The Kubernetes NCNs are the management nodes which are known as Kubernetes master nodes
+(ncn-mXXX) or Kubernetes worker nodes (ncn-wXXX). The only type of management node which is
```
Comment (mharding-hpe) [inline glossary.md:149]:
```suggestion
(`ncn-mXXX`) or Kubernetes worker nodes (`ncn-wXXX`). The only type of management node which is
```

File: glossary.md [hunk 2]
```diff
@@ -143,6 +143,12 @@ management VLAN is used for customer access and administration of hardware.
...
+excluded from this is the utility storage node (ncn-sXXX).
```
Comment (mharding-hpe) [inline glossary.md:150]:
```suggestion
excluded from this is the utility storage node (`ncn-sXXX`).
```

File: operations/spire/Restore_Spire_Postgres_without_a_Backup.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ event that spire-postgres databases cannot be restored from a backup.
    ncn# kubectl get pvc -n spire | grep spire-data-spire-server | awk '{print $1}' | xargs kubectl delete -n spire pvc
    ```
 
-3. Disable spire-agent on all of the Kubernetes NCNs and delete the join data.
+3. Disable spire-agent on all of the Kubernetes NCNs (all worker nodes and master nodes) and delete the join data.
```
Comment (mharding-hpe) [inline operations/spire/Restore_Spire_Postgres_without_a_Backup.md:24]:
```suggestion
3. Disable `spire-agent` on all of the Kubernetes NCNs (all worker nodes and master nodes) and delete the join data.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
I will see if I can resolve the merge conflicts and linting errors.

**Comment by mharding-hpe** [comment]
It appears that all of the originally proposed changes already exist in the 1.2 branch, so I'm closing this PR.

**Pull request closed without merging**